scripts/preinstall.js

```diff
@@ -20,12 +20,20 @@ if (+nodeVersion[0] < 18) {
   process.exit(1);
 }
 
-// Check for cargo
+// Check for rust
 try {
-  childProcess.execSync('cargo --version');
+  let rustVersion = childProcess.execSync('rustc --version');
+  const [major, minor, patch] = rustVersion.toString().split(' ')[1].split('.');
+  if (+major < 1 || (+major === 1 && +minor < 70)) {
+    console.log(`Found ${rustVersion}`);
+    console.error(
+      'Please make sure that your installed Rust version is greater than v1.70. You can update your installed Rust version with `rustup update`'
+    );
```

Comment:
Can we use the semver utility to check if the version is high enough? This isn't a super important area of the codebase but it'd be cleaner